.github/workflows/draft_aryn-sdk_release.yml

```diff
@@ -0,0 +1,39 @@
+name: Draft release for aryn-sdk
+
+on:
+  push:
+    tags:
+      - "sdk-v*"
+
+permissions:
+  contents: write
+
+jobs:
+  draft-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install poetry
+        run: pipx install poetry
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: "poetry"
+      - name: Install
+        run: poetry install
+        working-directory: lib/aryn-sdk
+      - name: build
+        run: poetry build
+        working-directory: lib/aryn-sdk
+      - name: Generate Changelog
+        run: |
+          git log --format="* %s by %aE" $(git describe --tags --abbrev=0 --match 'sdk-v*' \
+          --match 'v0.1.16' @^)..@ -- lib/aryn-sdk > ${{ github.workspace }}-CHANGELOG.txt
```

Comment:
Is this filtering the commits to only show those related to the SDK? That's kind of cool. I wonder if we should try to do that for the library release notes as well. I suppose this is also going to screw up the releases list on the right in Github, right? The version numbers will go back and forth between sdk releases and library releases?

Reply:
yep on both counts I think. I couldn't quite find a way around the release list on gh... would be interested in what our monorepo gurus think although my prediction is they'll say "oh we had internal tooling for that"...
For now I'm just assuming that a/ the vast majority of commits will be relevant to lib/sycamore so if a few sdk ones sneak into the notes, that's ok, and b/ we'll just do sycamore-ai releases after aryn-sdk ones so it'll always show up as the most recent

Reply:
Do we need to do a github release at all? I don't think the pypi release really depends on it, or at most it uses it as a way to get a specific commit hash, which we can do in a bunch of ways. 

I'm not /that/ worried about the gh release list, but if it's easy to avoid it might be worth it.

Reply:
So it looks to me that github really wants single release per repo [1]. Every repo I can find has a single "latest" release. It appears that the standard pattern is multiple pypi packages from a single repository [2]. I also found a github action from google to do releases [3] along with a ton of unrelated pages I looked at.

So given that, I would not do a github release for the library and only do a pypi release.

[1] https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
[2] https://stackoverflow.com/questions/54265944/python-multiple-packages-in-one-repository-or-one-package-per-repository
[3] https://github.com/googleapis/release-please-action